starter/script.js

```diff
@@ -1 +1,212 @@
-// put code here!
+//Mythical Beauty War
+
+//Goal is to get all the makeup, hair, and clothing neccessities for the Princess. And bring
+// them back so that she can go to THE red carpet event.
+
+//Items to get:
+//Elf workers gift = heels/shoes
+//Unicorn Rainbow throwup = lip sticks
+//Medusa hair = necklace
+//Dragon scales = dress
+//Fairy dust = shimmer and highlights
+//Mermaid hair = extentions or weave//Phoenix ashes = smokey eyes
+
+//First I need something to hold items: a dictionary/key:value pair.
+var backPack = {'Unicorn Rainbow Lipstick':0, 'Mermaid Weave':0, 'Dragon Scale Dress': 0,
+  'Medusa necklace':0, 'Stilletoes by ELF': 0, 'SmokeyEye by PHOENIX': 0, 'Fairy Face Shimmer': 0}
+
+function noAnswerLoop(reply){
+  while(reply === ""){
+    var reply = prompt("You must make a choice, " + userName);
+  }
+  return reply;
+};
+
+
+//The story begins.
+alert("The Millinnial Princess Pink Apple Everly Reignbeau \naka Princess PEAR of MacEphora Lusha Brown Land \nis getting ready for her first Golden Carpet event. \n\nShe has summoned you to her chambers.\n");
+
+//Get name from user
+var userName = prompt("Princess Pear: 'What is your name great warrior?'" , "Type Name.");
+
+//Princess Pear's Mission
+//If I console.log(firstReply); I do get whatever I put in. So the error is in the loop.
+alert("Princess PEAR: \n'Oh," + " "+ userName + " " + "all the things I have are so common, the other Princesses will surely have the same thing on. I need you to find me the rarest of the rare things that will make my beauty stand out above all the others. \n\nWill you help?'");
+var firstReply = prompt("Will you help the princess? \n\nType 'Y' for 'Yes, gladly, where do I start' \nOR 'N' for 'No, unless you fill my pockets with gold' \nOR 'B' for 'But, your beauty outshines the others, you could wear a trashbag.'");
+
+//How will user began journey?
+
+firstReply = noAnswerLoop(firstReply);
```

Comment:
Can you think of a way to write `noAnswerLoop` (and perhaps rename) so that you **do not** have to ask for an initial prompt and then also call `noAnswerLoop`?